raven-app/src/components/feature/channel-member-details/UserActions/UpdateAdminStatusButton.tsx

```diff
@@ -0,0 +1,53 @@
+import { Member } from '@/utils/channel/ChannelMembersProvider'
+import { BiCrown, BiSolidCrown } from 'react-icons/bi'
+import { useFrappeGetCall, useFrappeUpdateDoc } from 'frappe-react-sdk'
+import { toast } from 'sonner'
+import { getErrorMessage } from '@/components/layout/AlertBanner/ErrorBanner'
+
+interface UpdateAdminStatusButtonProps {
+    user: Member,
+    channelID: string,
+    updateMembers: () => void
+}
+
+export const UpdateAdminStatusButton = ({ user, channelID, updateMembers }: UpdateAdminStatusButtonProps) => {
+
+    const { updateDoc, loading: updatingMember, reset } = useFrappeUpdateDoc()
+
+    const { data: member } = useFrappeGetCall<{ message: { name: string } }>('frappe.client.get_value', {
+        doctype: "Raven Channel Member",
+        filters: JSON.stringify({ channel_id: channelID, user_id: user.name }),
+        fieldname: JSON.stringify(["name"])
+    }, undefined, {
+        revalidateOnFocus: false
+    })
+
+    const updateAdminStatus = async (admin: 1 | 0) => {
+        return updateDoc('Raven Channel Member', member?.message.name ?? '', {
+            is_admin: admin
+        }).then(() => {
+            toast.success('Member has been made an admin')
```

Comment:
A check needs to be done to understand whether we are removing or adding admin privileges to the Raven Channel member.

Alternatively, the toast message could read as `Channel Admins updated` to keep it generic